modules/aws-eks-managed-node-groups/locals.tf

```diff
@@ -9,14 +9,13 @@ locals {
     subnet_type              = "private"
     subnet_ids               = []
     release_version          = ""
+    force_update_version     = null
 
     desired_size    = "3"
     max_size        = "3"
     min_size        = "1"
     max_unavailable = "1"
-
-    disk_size = 50
-    disk_type = "gp3"
+    disk_size       = 50
```

Comment:
there should be `disk_size` or `block_device_mappings`, not both

Reply:
`disk_size` variable is used for Managed Node groups without Launch Templates.

Reply:
oh I see - this is like a set of "defaults". I would reconsider this approach - it expects users to know where to dive into the source code to find out where the default value is coming from